### DISCUSSION TITLE
⏳ Center bottom row element in Custom Loop Grids

### DISCUSSION THREAD

**Discussion by portrayaloflife** [elementor/elementor#24895] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Allow an option to center bottom row elements in custom loops if you're doing something like a 3-4+ column grid with multiple rows. An option for the last row to have the remaining single hanging element centered horizontally rather than aligning left. 

### Describe the solution you'd like.

Check box option in Custom Loops, for "center final row element"

### Describe alternatives you've considered.

Custom Coding plugin

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by micmiller-apptio** [comment]
Perhaps this could be accomplished by enabling spacing options for loop grids like there are for containers - start, end, space around, space between, etc.

**Comment by arsanet** [comment]
EXACTLY bro. This is one of the problems that can be easily solved by Elementor and has a great impact on the site view and listings. I had this problem and every time I fooled myself to deal with it, reducing the rows and then pagination!
If they add this item, it will be great.

**Comment by Daveden2** [comment]
Elementor is using CSS Grid for their loop grid. Unfortunately, there isn't a good way to align the last row to centre using grid.

**Comment by shimsim** [comment]
+1 for adding this functionality, as difficult as it might be due to the framework, its a feature that every 'good' loop/grid should have

**Comment by askwpgirl** [comment]
The reason for this not to center is due to Loop Grid using CSS Grid, ie display: grid. 

The beauty of CSS Grid is that it handles the grids perfectly. 

The only way to center the last row reasonably is to change this to display: flex. But that is not ideal for the reasons why CSS Grid is ideal. 

You can use CSS pseudo classes to control center leftover grid items:

https://css-irl.info/controlling-leftover-grid-items/
